GitCommands/Git/RevisionDiffProvider.cs

```diff
@@ -32,21 +44,53 @@ public sealed class RevisionDiffProvider : IRevisionDiffProvider
         /// <returns></returns>
         public string Get(string from, string to)
         {
+            return GetInternal(from, to);
+        }
+
+        /// <summary>
+        /// options to git-diff from GE arguments, including artificial commits
+        /// This is an instance class to not have static dependencies in GitModule
+        /// </summary>
+        /// <param name="from">The first revision</param>
+        /// <param name="to">The second "current" revision</param>
+        /// <param name="fileName">The file to compare</param>
+        /// <param name="oldFileName">The old name of the file</param>
+        /// <param name="isTracked">The file is tracked</param>
+        /// <returns></returns>
+        public string Get(string from, string to, string fileName, string oldFileName, bool isTracked)
+        {
+            return GetInternal(from, to, fileName, oldFileName, isTracked);
+        }
+
+        /// <summary>
+        /// options to git-diff from GE arguments, including artificial commits
+        /// This is an instance class to not have static dependencies in GitModule
```

Comment:
is this comment unnecessary? if so, why?

Reply:
The comments was to explain why the implementation is done like it is (was on your request)
What should be said?

Reply:
I don't think we need to explain why we do solid classes :) or why we don't have static dependencies.